### DISCUSSION TITLE
App icon for Wallpaperer

### DISCUSSION THREAD

**Discussion by ghost** [elrumo/macOS_Big_Sur_icons_replacements#1321] [Icon request] [answered]
All my icons got a nice icon except for this one, it's called [Wallpaperer](https://apps.apple.com/ca/app/wallpaperer/id1102248738?l=fr&mt=12)

**Comment by Mox4572** [comment, marked as answer]
![Wallpaperer](https://user-images.githubusercontent.com/80015221/141220936-55fe0e52-b493-4d51-bc6b-940ab21d149f.png)

**Reply by ghost** [reply]
Thanks a lot, this is the best looking one
